libvips/foreign/foreign.c

```diff
@@ -2458,6 +2458,11 @@ vips_webpload_buffer( void *buf, size_t len, VipsImage **out, ... )
  * Optional arguments:
  *
  * @Q: quality factor
+ * @lossless: enables lossless compression
+ * @preset: #VipsForeignWebpPreset choose lossy compression preset
+ * @smart_subsample: enables high quality chroma subsampling
+ * @near_lossless: use preprocessing in lossless mode (controlled by Q)
+ * @alphaQ: set alpha quality in lossless mode
  *
  * See also: vips_webpload(), vips_image_write_to_file().
```

Comment:
Could we add a note about the need to `g_free()` thre result? See `vips_pngsave_buffer()`.